apps/emqx_bridge_datalayers/mix.exs

```diff
@@ -0,0 +1,32 @@
+defmodule EMQXBridgeDatalayers.MixProject do
+  use Mix.Project
+  alias EMQXUmbrella.MixProject, as: UMP
+
+  def project do
+    [
+      app: :emqx_bridge_datalayers,
+      version: "0.1.0",
+      build_path: "../../_build",
+      erlc_options: UMP.erlc_options(),
+      erlc_paths: UMP.erlc_paths(),
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.14",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  def application do
+    [extra_applications: UMP.extra_applications()]
+  end
+
+  def deps() do
+    [
+      {:influxdb, github: "emqx/influxdb-client-erl", tag: "1.1.13"},
```

Comment:
Since this is (now) a shared dependency with `emqx_bridge_influxdb`, it would be best to move it to a `common_dep/1` clause in the root `mix.exs`, so we can centralized shared deps there.

Reply:
Thanks for your comments. Since I don't understand the mix compilation process, I'm unsure if my new commit understands what you mean.

Please help review it again.